pkg/azurecluster/validate_update.go

```diff
@@ -53,6 +53,11 @@ func (a *UpdateValidator) Validate(ctx context.Context, request *v1beta1.Admissi
 		return microerror.Maskf(errors.ParsingFailedError, "unable to parse AzureCluster CR: %v", err)
 	}
 
+	if !azureClusterNewCR.GetDeletionTimestamp().IsZero() {
```

Comment:
I'm not sure if it would be better to check both `azureClusterOldCR` and `azureClusterNewCR` , but the only situation where `azureClusterNewCR` would have `DeletionTimestamp` set and `azureClusterOldCR` wouldn't, would be when deleting the CR. With current code, in that case it would be validated one last time.

Thoughts?

Reply:
Yeah if we are deleting it we don't care about it being valid any more